lnrpc/lightning.proto

```diff
@@ -3002,6 +3002,16 @@ message Hop {
 
     // The payment metadata to send along with the payment to the payee.
     bytes metadata = 13;
+
+    // Blinding point is an optional blinding point included for introduction
+    // nodes in blinded paths.
+    bytes blinding_point = 14;
+
+    // Encrypted data is a receiver-produced blob of data that provides hops
```

Comment:
Q:does this also account for the case where the sender and receiver combine blinded paths to hide the introduction point for example, then part of the encrypted data would be generated by the sender?